### ISSUE TITLE
[Bug] SAF File picker does not exist on Android TV, Needs ability to use 3rd party file manager

### ISSUE THREAD

**Issue by Wave6677** [NeoApplications/Neo-Backup#881]
**Description**
The setup can not be completed as you can not pick a directory to save backups. This is due to Android/Google TV lacking com.android.documentsui and/or com.google.android.documentsui.
**Steps To Reproduce**
1.  Install
2. Grant permissions
3. Can not select backup directory due to "You do not have an app that can do this"
4. Can not procede

**Expected behavior**
Prompts to open third party file manager (Material Files, Fossify File Manager, Little File manager etc)

**Screenshots**
![image](https://github.com/user-attachments/assets/5c618d57-c3e1-473f-8611-06b080979cb6)

**System Information(please complete the following information):**

- Device: 
- Android Version: 12 
- ROM: Stock Rooted
- App's Version: 8.3.6

-----------------------------

**Additional Notes**

Installed from F-Droid
It appears other apps have implemented some sort of alternative picker 
https://github.com/TeamNewPipe/NewPipe/issues/7446
https://github.com/aniyomiorg/aniyomi/issues/1271

**Comment by machiav3lli** [comment]
Yeah, seems like Google/Android and the OEMs (also LineageOS) are not interested in providing such access. So the only maintainable way is the one that NP took.

**Comment by hg42** [comment]
with SAF DocumentsUI is required to grant access to a directory.

The only other possibility would be to use RootFile directly.
Michael suggested to add a simple input field for an alternate directory that is directly used with shadowRoot without using SAF at all. This could be done with a file:///... url as a value.
Either in a second pref that overrides the backupDir pref or by allowing editing of the url in backupDir.
May be I'll add this to devsettings and the appropriate code in shadowRoot to detect a file url.

In a longer run, I could also add a simple file picker, that can use StorageFile machinery together with shadowRoot.
It's just a list entering and leaving directories and an ok button.

**Comment by Tech33Blue** [comment]
I got similar problem with syncthing-fork on Google TV Chromecast running on android12, where when selecting "directory option" to browse the folder I wanted to sync, it told me "You do not have an app that can do this". I installed many file managers, but none solved the issue.
Eventually, I realized that if pressing on the gear icon next to directory with the Google TV remote, rather than the mouse, it worked, and it allowed me to browse and select a folder.
